contrib/spec/podman.spec.in

```diff
@@ -69,6 +69,7 @@ Recommends: container-selinux
 %else
 Requires: container-selinux
 %endif
+Recommends: slirp4netns
```

Comment:
Can you put this into the same block above.  So on RHEL it Requires else it recommends.

Reply:
It doesn't exist in RHEL.

Reply:
Well it will exist in RHEL8.  But I am fine with just moving it up with the ifdef block.

Reply:
I moved the line in the if branch, so it'll affect Fedora. Once it's packaged in RHEL, it can be added to the else branch.